docs/install.md

```diff
@@ -54,7 +54,19 @@ kubectl apply -k /path/to/apisix-ingress-controller/samples/deploy
 
 ## Helm
 
-// todo
+First of all, deploying the necessary CRDs and cluster resources like serviceaccount through the [base](../charts/base) chart.
```

Comment:
I think we do not need to deal with crd and deployment separately. And there are duplicate definitions in _helper.tpl.
Because it doesn’t matter if you execute it repeatedly, use helm upgrade when upgrading.

Reply:
Just reference Istio, separate base dependencies and apps.